string goal_name
---
float32[] coordinates
